.pre-commit-config-optional.yaml

```diff
@@ -1,6 +1,6 @@
 repos:
   - repo: https://github.com/tcort/markdown-link-check
-    rev: v3.10.3
+    rev: v3.11.0
```

Comment:
Wait for the bug to be fixed: https://github.com/autowarefoundation/autoware/pull/3383

Reply:
Fixed in v3.11.1.